### ISSUE TITLE
Flutter_driver from sdk vs angel3 framework dependency clash

### ISSUE THREAD

**Issue by hicnar** [dukefirehawk/angel#103]
![Screenshot 2023-06-10 at 13 30 40](https://github.com/dukefirehawk/angel/assets/1816194/6fb5f122-87fa-4346-85b7-08e4c1433586)
I may be wrong, but perhaps it would be prudent to wait till the sdk guys who maintain the flutter_driver update the 'file' dependency to 7.0.0 and then make angel3_framework depend on that? Especially that change of the major part (from 6 -> 7 ) may indicate some breaking changes in the package.

**Comment by dukefirehawk** [comment]
Thank you for feedback. This is an interesting use case. Thus far we have only updated the dependencies based on Dart SDK releases. Typically when a major dart SDK version is released, it is common practice to release a new version with all the dependencies updated to the latest versions. This will allow downstream packages that depend on them to be upgraded too. That said, it is not a good idea to held back the release due to packages that are not used by the framework.  In fact, we do waited for a number of packages that we needed to be available before this release.
